kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/documentation/links/NameAndUrlPopoverViewImpl.less

```
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+[data-i18n-prefix="NameAndUrlPopoverViewImpl."] {
+
+  #popover-container {
+    display: inline;
+  }
+
+  #popover-container .popover {
+    min-width: 300px;
+  }
+
+  .kie-dmn-name-and-url-type-container #kieName {
+    width: 100%;
+  }
+
+  .kie-dmn-name-and-url-type-container #kieDataType {
+    width: 100%;
+  }
+
+  .attachment-tip{
```

I think we have a format issue here:
```suggestion
  .attachment-tip {
```